client/app/services/parameters/NumberParameter.js

```diff
@@ -0,0 +1,20 @@
+import { toNumber, isNull } from 'lodash';
+import { Parameter } from '.';
+
+class NumberParameter extends Parameter {
+  constructor(parameter, parentQueryId) {
+    super(parameter, parentQueryId);
+    this.setValue(parameter.value);
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  normalizeValue(value) {
+    if (isNull(value)) {
+      return null;
+    }
+    const normalizedValue = toNumber(value);
```

Comment:
`toNumber` converts empty strings to 0, but I think it should be normalized to `null` instead.
How about:
```js
import { toNumber, trim } from 'lodash';

normalizeValue(value) {
    if (!trim(value)) {
      return null;
    }
    ...
}
```